tensorflow/lite/kernels/split_v_test.cc

```diff
@@ -12,8 +12,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-#include <initializer_list>
```

Comment:
I think the previous ordering is better.

Reply:
@haozha111 : This was reported in static check, so modified, still need to revert?

Reply:
Yes, we usually import headers in the following order:
1) C++ STL
2) Google-specific libraries (testing, etc)
3) Everything under tensorflow/

So please revert those. Thanks!